CommonTools/CandAlgos/plugins/CandReducer.cc

```diff
@@ -16,20 +16,20 @@
  * $Id: CandReducer.cc,v 1.3 2009/09/27 22:26:55 hegner Exp $
  *
  */
-#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
 #include "FWCore/Utilities/interface/InputTag.h"
 #include "DataFormats/Candidate/interface/Candidate.h"
 
-class CandReducer : public edm::EDProducer {
+class CandReducer : public edm::global::EDProducer<> {

```

Comment:
for my understanding, why global and not stream?

Reply:
I have been under the impression that when there are no internal state to update, using `global` is preferable because requires less memory, but @makortel can correct me if I am wrong

Reply:
Yes, if it is possible to make a module `global` this is the most efficient way to run it multithreaded